### PR TITLE
Wadeea js2 homework week1

### PR DESCRIPTION
**Pull request by WadeeaKiwan**

### CODE REVIEW

File: Week1/homework/app.js [hunk 1]
```diff
@@ -1,11 +1,170 @@
...
+      const images = document.createElement('img');
+      // 1.6 sources to each of the images are added
+      images.setAttribute('src', bookImages[bookInfo[i]]);
+      // 1.6 alts to each of the images are added
+      images.setAttribute('alt', bookTitles[i]);
+      listItem.appendChild(images);
+    }
+  }
+
+  function main() {
```
Comment (rohni) [inline Week1/homework/app.js]: Nice touch creating a main function

File: Week1/homework/app.js [hunk 2]
```diff
@@ -1,11 +1,170 @@
...
+    bookDetails(booksList);
+    const p = document.createElement('p');
+    /* cSpell:disable */
+    p.innerText = '© Wadeea Kiwan 2019';
+    /* cSpell:enable */
+    p.className = 'footer';
+    root.appendChild(p);
+  }
+
+  window.addEventListener('load', main);
```
Comment (rohni) [inline Week1/homework/app.js]: And loading everything once the page is rendered.  Very nice!

File: Week1/homework/app.js [hunk 3]
```diff
@@ -1,11 +1,170 @@
...
+      const authors = document.createElement('p');
+      authors.innerText = bookInformation[bookInfo[i]].author;
+      listItem.appendChild(authors);
+      const languages = document.createElement('p');
+      languages.innerText = bookInformation[bookInfo[i]].language;
+      listItem.appendChild(languages);
+      const publishing = document.createElement('p');
+      publishing.innerText = bookInformation[bookInfo[i]].published;
+      listItem.appendChild(publishing);
+      const images = document.createElement('img');
```
Comment (rohni) [inline Week1/homework/app.js]: According to the spec, the book covers should only be added after the whole list is built, but finding the elements and adding the book cover dynamically.  (Yes it is a silly restriction, but it is more about the knowledge needed to make it happen than anything else 😉)

File: Week1/homework/app.js [hunk 4]
```diff
@@ -1,11 +1,170 @@
...
+    little_women: 'images/little_women.jpg',
+    ophelia: 'images/ophelia.jpg',
+  };
+
+  // 1.6 Beautify your html page with css (use the style.css file for that), add sources and alts to each of the images.
+
+  // 1.5 - 1.8
+  function bookDetails(booksList) {
+    for (let i = 0; i < bookTitles.length; i++) {
+      const listItem = document.createElement('li');
```
Comment (rohni) [inline Week1/homework/app.js]: Personally, I would have broken this part up into one or more functions, and then called the functions within the look passing in the book data. 
This works, it is just a maintainability issue, and personal preference.

File: Week1/homework/style.css [hunk 5]
```diff
@@ -1 +1,81 @@
-/* add your styling here */
\ No newline at end of file
+body {
```
Comment (rohni) [inline Week1/homework/style.css]: Nice work with the CSS.  Some really nice extras in here. 👍

File: Week1/homework/app.js (hunk 4 above)
Comment (WadeeaKiwan) [reply Week1/homework/app.js]: Many thanks, I'll take your feedback into consideration while improving this homework and next works.

File: Week1/homework/app.js (hunk 3 above)
Comment (WadeeaKiwan) [reply Week1/homework/app.js]: I'll improve this one also.

File: Week1/homework/style.css (hunk 5 above)
Comment (WadeeaKiwan) [reply Week1/homework/style.css]: 🙂

**Pull request closed without merging**